### DISCUSSION TITLE
Include Nextcloud / Seafile

### DISCUSSION THREAD

**Discussion by CaldeiraG** [tteck/Proxmox#207] [Ideas]
Hello, great work so far on the scripts, have been using some LXCs for a while.

I was wondering if a Nextcloud or alternatively Seafile LXC script could be created.

Thanks in advance.

**Comment by nacho-domatix** [comment]
Best opensource "file sync" solutions indeed. I used to love Seafile before more than Nextcloud, but the fact now they have "Enterprise edition" and not giving the community edition basic features like garbage collector (you need to reboot the server), makes me believe that Nextcloud is the best choice.
Also the UI and features of Seafile are basically stuck in 2015. And the fact that using the client in a non-debian distro is a pita. I'm still using it tho, but not for longer.

**Reply by nacho-domatix** [reply]
https://github.com/nextcloud/nextcloudpi/issues/1553#issuecomment-1254842166

**Reply by tteck** [reply]
@nacho-domatix, NextCloudPi Done!

**Reply by nacho-domatix** [reply]
Amazing!!!

Many thanks! I was seeking for this so long time! :)

**Reply by CaldeiraG** [reply]
Great news! :)
